### PR TITLE
Dissolves Cargonia

### PR DESCRIPTION
**Pull request by Lorwp**
🆑 lorwp
del: The Quartermaster is no longer a Head of Staff. Honk
/:cl:

Honk. Reverts #582 via 650019a3ebc4834865c912552c231206beb2561b

### CODE REVIEW

**Comment by AffectedArc07** [comment]
And the PR that gives them the keycard authentication thing just got merged

**Comment by Lorwp** [comment]
[Closed](https://github.com/OracleStation/OracleStation/pull/759#issuecomment-378725106)

File: strings/sillytips.txt [hunk 1]
```diff
@@ -15,7 +15,7 @@ The wizard is supposed to be extremely strong in one on one combat, stop getting
 Sometimes a round will just be a bust. C'est la vie.
 This is a game that is constantly being developed for. Expect things to be added, removed, fixed, and broken on a daily basis.
 It's fun to try and predict the round type from the tip of the round message.
-The quartermaster IS a head of staff.
+The quartermaster is not a head of staff and will never be one again.
```
Comment (SamHPurp) [inline strings/sillytips.txt]: I'm glad you did this.

Comment (Lorwp) [reply strings/sillytips.txt]:
```dm
//TODO: Add another 'again' for every round where the QM tries to be a Head of Staff
```

**Pull request merged**